docs/ServiceNow-batchsource.md

```diff
@@ -49,6 +49,12 @@ ignored if the Mode is set to `Reporting`.
 
 **Password**: The password for ServiceNow Instance.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Table Names information will be provided by the connection.
```

Comment:
I think it should be Table Name not Names since the plugin reads from one table? Also, the order of the properties should match the order they appear in the UI.

Reply:
Done